tripper/datadoc/clitool.py

```diff
@@ -45,6 +46,21 @@ def subcommand_add(ts, args):
         ts.serialize(args.dump, format=args.format)
 
 
+def subcommand_delete(ts, args):
+    """Subcommand for removing matching entries in the triplestore."""
+    criterias = {}
+    regex = {}
+    if args.criteria:
+        for crit in args.criteria:
+            if "=~" in crit:
+                key, value = crit.split("=~", 1)
+                regex[key] = value
+            else:
```

Comment:
Example of strange codecov report:
- `subcommand_delete()` is definitely called by `test_delete()` in test_datadoc_dli.py. So line 51-53 must be covered.
- Hwy can line 58 be covered when line 51-57 isn't?